### DISCUSSION TITLE
Access results from Insights Panel in an custom endpoint

### DISCUSSION THREAD

**Discussion by co2nunes** [directus/directus#18837] [Feature Requests]
### Summary

I think it is a new feature, or I couldn't find any other information regarding the Insights API.
I created a couple of dashboards but when I GET a Request from Insomnia, for instance, the totals of each Panel return nothing

### Basic Example

<img width="884" alt="Captura de ecrã 2023-06-07, às 17 43 55" src="https://github.com/directus/directus/assets/127744439/5a0e7ada-d9ec-4d74-8606-13017b0ff722">

### Motivation

I just wanted to access 30, and 8 from each Panel in the Dashboards.

GET Request to panels/3c183ade-0c64-436c-ad07-f6fd8def02da

Returns:

```
{
	"data": {
		"id": "2c583dae-9c64-436c-ad07-f6fd8def02db",
		"dashboard": "---------",
		"name": "Total de Utilizadores",
		"icon": null,
		"color": null,
		"show_header": true,
		"note": null,
		"type": "metric",
		"position_x": 1,
		"position_y": 1,
		"width": 14,
		"height": 6,
		"options": {
			"type": "integer",
			"inter": "input",
			"collection": "directus_users",
			"sortField": "first_name",
			"field": "id",
			"function": "count",
			"suffix": " Users",
			"abbreviate": true
		},
		"date_created": "2023-06-07T14:02:08.900Z",
		"user_created": "----"
	}
}

```

### Detailed Design

If we have to do it on the server side in bare code to expose these data to React, for instance, it doesn't make much sense to me.

### Must Have:
- Expose results from Insights Panel into the API

### Drawbacks

shouldn't break any change, it's optional.
A checkbox to expose, there is already a granular permission system builtin =)
implementation should be fairly easy, I use Docker though.

### Alternatives

Bare code, only code.

### Adoption Strategy

I think it's a plus if you can't expose Directus to your client, I don't think it would break to add a checkbox to share the result with a developer.

**Comment by pedromcipriano** [comment]
I would be nice to have this feature.

**Reply by rijkvanzanten** [reply]
Make sure to give it an upvote if you haven't already!

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
